README.md

```diff
@@ -17,95 +15,72 @@ installation] document.
 You also need a running Zonemaster-Backend component. Please see the
 [Zonemaster-Backend installation] document.
 
-##### Configuration
+### Installation
+
+Follow the detailed [installation instructions].
+
+### Configuration
 
 The configuration instructions for the backend can be found in the [Backend
 configuration] document.
 
 The configuration instructions for the GUI can be found in the [GUI
 configuration] document.
 
+### Documentation
+
+There is a [public documentation]. Some more specific documents can be found in
+the [docs directory](docs/).
 
 ### Contribution
 
 This project was generate with Angular-CLI 1.6.8 and then updating.
```

Comment:
What does the sentence mean?

Reply:
It is unclear to me as well, I shortened it to "This project was generated with Angular-CLI 1.6.8.", even though I don't know how relevant this information is.